tkg/managementcomponents/helper.go

```diff
@@ -168,16 +168,38 @@ func convertToBool(config interface{}) bool {
 	return false
 }
 
-// autofillAkoOperatorConfig autofills empty fields in AkoOperatorConfig
+// convertAVILabels converts config into string type, and return empty string if config is not set
+func convertAVILabels(config interface{}) (string, error) {
+	if config != nil {
+		switch config.(type) {
+		case string:
+			return config.(string), nil
+		default:
```

Comment:
Hm, would it always be string type and this will never be reached?
maybe we could just do val := config.(string)
And the runtime panic will happen if it's not string type

Reply:
Actually not, it could be: 
- string `AVI_LABELS: '{"foo":"bar"}`, 
- map: 
```yaml
AVI_LABELS: |
    foo: bar
```
So it can be touched.